<launch>
  <group ns="my_stereo">
    <node name="left" pkg="usb_cam" type="usb_cam_node" output="screen" >
      <param name="video_device" value="/dev/video1" />
      <param name="image_width" value="640" />
      <param name="image_height" value="480" />
      <param name="pixel_format" value="yuyv" />
      <param name="camera_frame_id" value="usb_cam_left" />
      <param name="camera_name" value="left_cam" />
      <param name="framerate" value="30" />
      <param name="camera_info_url" value="package://stereo_vision/launch/left_cam.yaml" />
      <param name="io_method" value="mmap"/>
    </node>
    <node name="image_view_left" pkg="image_view" type="image_view" respawn="false" output="screen">
      <remap from="image" to="/my_stereo/left/image_raw"/>
      <param name="autosize" value="true" />
    </node>

    <node name="right" pkg="usb_cam" type="usb_cam_node" output="screen" >
      <param name="video_device" value="/dev/video0" />
      <param name="image_width" value="640" />
      <param name="image_height" value="480" />
      <param name="pixel_format" value="yuyv" />
      <param name="camera_frame_id" value="usb_cam_right" />
      <param name="camera_name" value="right_cam" />
      <param name="framerate" value="30" />
      <param name="camera_info_url" value="package://stereo_vision/launch/right_cam.yaml" />
      <param name="io_method" value="mmap"/>
    </node>
    <node name="image_view_right" pkg="image_view" type="image_view" respawn="false" output="screen">
      <remap from="image" to="/my_stereo/right/image_raw"/>
      <param name="autosize" value="true" />
    </node>
  </group>

</launch>
